README.md

```diff
@@ -54,16 +54,17 @@ You can typically configure your environment by editing `~/.zshrc` and adding th
 
 ```
 export PATH=$PATH:~/Library/Android/sdk/platform-tools/
+export PATH=$PATH:~/path_to_ninja_binary/
```

Comment:
Maybe append a comment to this line: `# Only for a manual installation of Ninja rather than using a package manager.